app/routes/events/view/speakers/edit.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+  titleToken(model) {
+    var speaker_name = model.get('name');
+    return this.get('l10n').t('Edit Speaker-'.concat(speaker_name));
```

Comment:
This is a dynamic string. Should not pass in directly. See https://github.com/Cropster/ember-l10n#singular-translations

Reply:
@niranjan94 Does the speaker's name at all needs to be passed for translation 😅 ?